cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -328,10 +462,69 @@ func (o *CreateOptions) Complete(clientGetter util.ClientGetter, cmd *cobra.Comm
 		}
 	}
 
-	if cmd.Flags().Changed(podRunningTimeout) && o.PodRunningTimeout <= 0 {
+	if cmd.Flags().Changed(podRunningTimeoutFlagName) && o.PodRunningTimeout <= 0 {
 		return errors.New("--pod-running-timeout must be higher than zero")
 	}
 
+	if !validateMutuallyExclusiveFlags(
+		o.SlurmFlagSet.Changed(memPerTaskFlagName),
+		o.SlurmFlagSet.Changed(memPerCPUFlagName),
+		o.SlurmFlagSet.Changed(memPerGPUFlagName)) {
+		return errors.New("the --mem-per-task, --mem-per-cpu and --mem-per-gpu options are mutually exclusive")
```

Comment:
I would like to use the same message as we have on cobra:

"if any flags in the group [mem-per-task mem-per-cpu mem-per-gpu] are set none of the others can be; [mem-per-task mem-per-cpu mem-per-gpu] were all set"

And also we can generate it on `validateMutuallyExclusiveFlags`. WDYT?